plugin/pkg/namespacedcloudprofile/validator/validator_suite_test.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
2024.
You could also run `make add-license-headers` after removing the old header

Reply:
This is one thing I am curious about. Should license headers only say 2024 in new files or should they also be updated in changes files?

Reply:
We typically do it for only new files. The command also does the same.

Reply:
Thanks for the Info. Will update the license headers.

Reply:
Fixed license headers in https://github.com/gardener/gardener/pull/9440/commits/16038f43eb24639a81a7a6dbaaf5ba554caa65a9.